### PR TITLE
“final”

### PR DESCRIPTION
**Pull request by Tarasov123555**
- [DEMO LINK](https://Tarasov123555.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Tarasov123555.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  height: 19px;
+  width: 19px;
+  border: none;
+  background-image: url(./images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-color: transparent;
+}
+
+.searchIconSmall {
```
Comment (Viktor-Kost) [inline src/style.css:67]: You have a lot of repeatable styles. 
So you need to refactor you code. I will give you an EXAMPLE, how to do it

Following the BEM-methodology - use the `block` selector for the FIRST form (`class="form"`), where you apply common form styles

```
.form {
  margin: 20px 0;
  border: solid 1px #e1e7ed;
  border-radius: 4px;
  height: 70px;
  position: relative;
  margin-top: 20px;
}
```

And then you apply `modifier` to the SECOND form (`class="form form--small"`) that have ONLY differences with base class form
```
.form--small {
  height: 42px;
  border-radius: 4px;
}
```
So in this way you follow DRY (Don't repeat yourself) principle
Apply this principe all over the styles

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+.searchInputSmall {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.searchInput::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
```
Comment (Viktor-Kost) [inline src/style.css:49]: create variables for repeated colors

File: src/index.html [hunk 3]
```diff
@@ -2,15 +2,43 @@
...
+      data-qa="big"
+    >
+      <label for="searchInputBig"></label>
+      <input type="text"
+        id="searchInputBig"
+        class="searchInput"
+        name="searchInput"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress" required
+      >
```
Comment (Viktor-Kost) [inline src/index.html:24]: you can include `<input>` into `<label>` to avoid `for`/`id`

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Comment by Tarasov123555** [comment]
- [DEMO LINK](https://Tarasov123555.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Tarasov123555.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/index.html (hunk 3 above)
Comment (vlad-demchuk) [inline src/index.html:24]:
```suggestion
      <input
        type="text"
        id="searchInputBig"
        class="searchInput"
        name="searchInput"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress" 
        required
      >
```

Comment (vlad-demchuk) [reply src/index.html:24]: fix the previous comments
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3121/files#r1145215078

File: src/style.css (hunk 2 above)
Comment (vlad-demchuk) [reply src/style.css:49]: the same 
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3121/files#r1145213280

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
if you have any questions, just feel free to ask in `fe_chat`

File: src/index.html [hunk 4]
```diff
@@ -2,15 +2,47 @@
...
+
+    <form
+      class="searchForm searchFormSmall"
+      action="/"
+      method="get"
+      data-qa="small"
+    >
+      <label>
+        <input type="text"
+        name="searchInputSmall"
```
Comment (YegorKorenkov) [inline src/index.html:38]: Please don't forget check your formatting(should be 2-space indentation)
image

**Review by YegorKorenkov — APPROVED** [review]
Nice work!

**Pull request closed without merging**